i18n/extract.py

```diff
@@ -316,9 +316,9 @@ def fix_metadata(pofile):
         'Last-Translator': '',
         'Language-Team': 'openedx-translation <openedx-translation@googlegroups.com>',
         'Plural-Forms': 'nplurals=2; plural=(n != 1);',
+        'PO-Revision-Date': '1970-01-01 00:00:00.000000',
```

Comment:
Please add `POT-Creation-Date` and put it on the top -- if the order matters.

Also, some tools may check for [timestamp epoch 0](https://www.unixtimestamp.com/).

Use any date you like that's relatively recent.

Reply:
Done